Documentation/spelling_wordlist.txt

```diff
@@ -153,6 +154,7 @@ Zannoni
 Zhou
 allocator
 allocators
+amd
```

Comment:
Is this avoiding the documentation breakage for using the word `amd64` ?

Reply:
yes!